public/scripts/raneto.js

```diff
@@ -18,8 +18,7 @@
       $(".content table").addClass("table");
 
       // FitVids
-      $(".content").fitVids();
-
+      fitVids();
```

Comment:
This is the vanilla version which is based on the same idea. I used it because the jQuery version isn't available in npm. 

It's worth noting that jQuery moved their plugin repository to npm (see https://plugins.jquery.com)

Project files:
https://www.npmjs.com/package/fitvids
https://github.com/rosszurowski/vanilla-fitvids